Code/C++/Postfixexpression.cpp

```diff
@@ -0,0 +1,101 @@
+//Evaluation of Postfix expression
```

Comment:
add a brief description of the algorithm.

Reply:
Hi 
I have added the description of the algorithm. Pls review it and if any further changes are required I would be happy to do them.